<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hamza</class>
 <widget class="QDialog" name="hamza">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_page4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>670</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  background-color: #87CEEB; /* blue sky color */
  border: 2px solid gray;
  border-radius: 5px;
  padding: 5px;
}

QLabel {
  color: white;
  font-size: 14px;
}

QLineEdit {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  padding: 2px;
}

QPushButton {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  color: gray;
  padding: 5px;
  font-size: 14px;
}

QPushButton:hover {
  background-color: gray;
  color: white;
}
</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>1051</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">$QStackedWidget {
  background-color: #87CEEB
}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>971</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
  background-color: #87CEEB
}</string>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>40</y>
        <width>451</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    font-size: 25px;
}

QTableView {
    font-size: 20px;
}
</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>421</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QGroupBox {
  background-color: #87CEEB; /* blue sky color */
  border: 2px solid gray;
  border-radius: 5px;
  padding: 5px;
}

QLabel {
  color: white;
  font-size: 14px;
}

QLineEdit {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  padding: 2px;
}

QPushButton {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  color: gray;
  padding: 5px;
  font-size: 14px;
}

QPushButton:hover {
  background-color: gray;
  color: white;
}
</string>
      </property>
      <property name="title">
       <string>Veuillez remplir les informations suivantes:</string>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit_etat">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>150</y>
         <width>261</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPlainTextEdit {
    font-size: 25px; /* Change the font size to 16 pixels */
}

QTableView {
    font-size: 25px; /* Change the font size to 14 pixels */
}
</string>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_matricule">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: blue;
    color: white;
}

QComboBox QAbstractItemView {
    background-color: blue;
    color: white;
}

QComboBox QLabel {
    color: white;
}

QComboBox QLineEdit {
    background-color: white;
    color: black;
}
QLineEdit {
    font-size: 25px; /* Change the font size to 16 pixels */
}

QTableView {
    font-size: 25px; /* Change the font size to 14 pixels */
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ajouter">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>240</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_modele">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    font-size: 25px; /* Change the font size to 16 pixels */
}

QTableView {
    font-size: 14px; /* Change the font size to 14 pixels */
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Modele:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_modifier">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>240</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_type">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Voitures</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Camions</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Motocyclettes et scooters</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tracteurs</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>201</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Type de vehicule:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>629</x>
       <y>940</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>950</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1001</width>
       <height>571</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
  background-color: #87CEEB
}</string>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>471</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    font-size: 25px; /* Change the font size to 16 pixels */
}

QTableView {
    font-size: 20px; /* Change the font size to 14 pixels */
}
</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>271</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
  background-color: #87CEEB; /* blue sky color */
  border: 2px solid gray;
  border-radius: 5px;
  padding: 5px;
}

QLabel {
  color: white;
  font-size: 14px;
}

QLineEdit {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  padding: 2px;
}

QPushButton {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  color: gray;
  padding: 5px;
  font-size: 14px;
}

QPushButton:hover {
  background-color: gray;
  color: white;
}
</string>
      </property>
      <property name="title">
       <string>trier par modele</string>
      </property>
      <widget class="QPushButton" name="pushButton_tri_modeleAZ">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-image: url(:/button (1).png);
}</string>
       </property>
       <property name="text">
        <string>A-Z</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tri_modeleZA">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-image: url(:/button (1).png);
}</string>
       </property>
       <property name="text">
        <string>Z-A</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>210</y>
        <width>271</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
  background-color: #87CEEB; /* blue sky color */
  border: 2px solid gray;
  border-radius: 5px;
  padding: 5px;
}

QLabel {
  color: white;
  font-size: 14px;
}

QLineEdit {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  padding: 2px;
}

QPushButton {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  color: gray;
  padding: 5px;
  font-size: 14px;
}

QPushButton:hover {
  background-color: gray;
  color: white;
}
</string>
      </property>
      <property name="title">
       <string>supprimer</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_matriculeSupprimer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 63 14pt &quot;Montserrat SemiBold&quot;;
QLineEdit {
    font-size: 25px; /* Change the font size to 16 pixels */
}

QTableView {
    font-size: 25px; /* Change the font size to 14 pixels */
}</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>35</y>
         <width>401</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
  background-color: #87CEEB; /* blue sky color */
  border: 2px solid gray;
  border-radius: 5px;
  padding: 5px;
}

QLabel {
  color: white;
  font-size: 14px;
}

QLineEdit {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  padding: 2px;
}

QPushButton {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  color: gray;
  padding: 5px;
  font-size: 14px;
}

QPushButton:hover {
  background-color: gray;
  color: white;
}
</string>
       </property>
       <property name="text">
        <string>MATRICULE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supprimer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>461</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
  background-color: #87CEEB; /* blue sky color */
  border: 2px solid gray;
  border-radius: 5px;
  padding: 5px;
}

QLabel {
  color: white;
  font-size: 14px;
}

QLineEdit {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  padding: 2px;
}

QPushButton {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  color: gray;
  padding: 5px;
  font-size: 14px;
}

QPushButton:hover {
  background-color: gray;
  color: white;
}
</string>
      </property>
      <property name="title">
       <string>Recherche</string>
      </property>
      <widget class="QPushButton" name="pushButton_chercher">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>20</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-image: url(:/button (1).png);
}</string>
       </property>
       <property name="text">
        <string>chercher par matricule </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_recherche">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    font-size: 25px; /* Change the font size to 16 pixels */
}

QTableView {
    font-size: 25px; /* Change the font size to 14 pixels */
}
</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_pdf">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>490</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
  background-color: #87CEEB; /* blue sky color */
  border: 2px solid gray;
  border-radius: 5px;
  padding: 5px;
}

QLabel {
  color: white;
  font-size: 14px;
}

QLineEdit {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  padding: 2px;
}

QPushButton {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  color: gray;
  padding: 5px;
  font-size: 14px;
}

QPushButton:hover {
  background-color: gray;
  color: white;
}
</string>
      </property>
      <property name="text">
       <string>exporter en PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stat_2">
      <property name="geometry">
       <rect>
        <x>582</x>
        <y>470</y>
        <width>261</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>981</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
  background-color: #87CEEB
}</string>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QLabel" name="label_pic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>441</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>                          l'image selectionné</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browseimage">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
  background-color: #b3d9ff; /* Light blue color */
}</string>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pic_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>afficher l'image</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_Image_List_From_Database">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>90</y>
        <width>511</width>
        <height>411</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>220</number>
      </attribute>
      <column>
       <property name="text">
        <string>Nouvelle colonne</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>image</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertion image</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>1011</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
  background-color: #87CEEB
}</string>
     </property>
     <property name="title">
      <string>stat</string>
     </property>
     <widget class="QPushButton" name="pushButton_stat">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_temp">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>210</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher temperature </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_temp">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>170</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>entrer la matricule :</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1031</width>
       <height>581</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
  background-color: #87CEEB
}</string>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>27</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse image</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_page3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>670</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  background-color: #87CEEB; /* blue sky color */
  border: 2px solid gray;
  border-radius: 5px;
  padding: 5px;
}

QLabel {
  color: white;
  font-size: 14px;
}

QLineEdit {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  padding: 2px;
}

QPushButton {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  color: gray;
  padding: 5px;
  font-size: 14px;
}

QPushButton:hover {
  background-color: gray;
  color: white;
}
</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_page2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>670</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  background-color: #87CEEB; /* blue sky color */
  border: 2px solid gray;
  border-radius: 5px;
  padding: 5px;
}

QLabel {
  color: white;
  font-size: 14px;
}

QLineEdit {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  padding: 2px;
}

QPushButton {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  color: gray;
  padding: 5px;
  font-size: 14px;
}

QPushButton:hover {
  background-color: gray;
  color: white;
}
</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_page1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>670</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  background-color: #87CEEB; /* blue sky color */
  border: 2px solid gray;
  border-radius: 5px;
  padding: 5px;
}

QLabel {
  color: white;
  font-size: 14px;
}

QLineEdit {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  padding: 2px;
}

QPushButton {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  color: gray;
  padding: 5px;
  font-size: 14px;
}

QPushButton:hover {
  background-color: gray;
  color: white;
}

</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>630</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Date and Time</string>
   </property>
   <widget class="QLabel" name="Digital_clock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CLOCK</string>
    </property>
   </widget>
   <widget class="QLabel" name="DATE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DATE</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_quitter">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>670</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  background-color: #87CEEB; /* blue sky color */
  border: 2px solid gray;
  border-radius: 5px;
  padding: 5px;
}

QLabel {
  color: white;
  font-size: 14px;
}

QLineEdit {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  padding: 2px;
}

QPushButton {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  color: gray;
  padding: 5px;
  font-size: 14px;
}

QPushButton:hover {
  background-color: gray;
  color: white;
}
</string>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
